pyseq.py

```diff
@@ -592,6 +620,116 @@ def _get_missing(self):
 
         return missing
 
+class multiViewSequence(Sequence):
+
+    def __init__(self, items, **kwargs):
+        """
+        multiViewSequence
+        sequence container 
+        items are items from a sequence object egg from a found left view seq[:]
+        kwargs get set as an attribute for each key 
+        {'left': sequence object left, 'right': sequence object}
+        ==> self.left = sequence object left
+            self.right = sequence.object right
+        to iterate through there is a views list
+        for view in seq.views:
+            viewSeq = getattr(seq,view)
+        all format functions on seq only return a %V or %v in the head or tail, depending on the view position
+        """
+        
+        super(multiViewSequence, self).__init__([Item(items.pop(0))])
```

Comment:
don't you just want to do:
    super(multiViewSequence, self).__init__(items)
    # skip the while loop
    for key in kwargs:
        setattr...

Cause Sequence is going to run the while loop if you give it all the items, and you don't seem to be doing anything different.